Description: Change Makefile to fit debian policy
Author: Luca Bedogni <me@lucabedogni.it>
Last-Update: 2015-07-12

--- a/src/Makefile
+++ b/src/Makefile
@@ -3,7 +3,7 @@
 LIBDIR += -L/usr/X11R6/lib
 LDFLAGS += -lXpm -lXext -lX11 -lm -s
 #CFLAGS = -g #-DMONDEBUG
-CFLAGS += -Wall -O3 -m486 -fomit-frame-pointer -I/usr/X11R6/include #-DHI_INTS #undefine HI_INTS if your x86 SMP or alpha
+CFLAGS += -Wall -O3 -fomit-frame-pointer -I/usr/X11R6/include #-DHI_INTS #undefine HI_INTS if your x86 SMP or alpha
 
 BIN = wmsysmon
 OBJS =	wmgeneral.o \
@@ -17,5 +17,6 @@
 clean:
 	rm -f $(OBJS) $(BIN)
 
-install:
-	install -g root -o root $(BIN) /usr/local/bin
+install: $(BIN)
+	mkdir -p $(DESTDIR)/usr/bin
+	install -g root -o root $(BIN) $(DESTDIR)/usr/bin
